application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -185,20 +201,35 @@ private void deleteTag(@NotNull CommandInteraction event) {
      *
      * @param requiredTagStatus the required status of the tag
      * @param idAction the action to perform on the id
-     * @param actionVerb the verb describing the executed action, i.e. <i>edited</i> or
-     *        <i>created</i>, will be displayed in the message send to the user
      * @param event the event to send messages with, it must have an {@code id} option set
+     * @param subcommand the executed subcommand
```

Comment:
`executed` is past tense but the `handleAction` is the one executing it. So its not over yet, you need to use present-tense. "the subcommand to be executed" or "the subcommand to execute"

Reply:
done.